src/oneid/oneid-ecs-core/src/main/resources/application.properties

```diff
@@ -15,4 +15,19 @@ quarkus.log.category."it.pagopa.oneid".level=DEBUG
 %dev.quarkus.dynamodb.aws.credentials.static-provider.secret-access-key=testsecret
 #Prod
 %prod.quarkus.dynamodb.aws.region=eu-south-1
-%prod.quarkus.dynamodb.aws.credentials.type=default
\ No newline at end of file
+%prod.quarkus.dynamodb.aws.credentials.type=default
+
+# quinoa
+quarkus.quinoa.dev-server.port=3000
+quarkus.quinoa.build-dir=build
+quarkus.quinoa.package-manager=yarn
+quarkus.quinoa.package-manager-install=true
+quarkus.quinoa.package-manager-install.node-version=20.14.0
+quarkus.quinoa.package-manager-install.yarn-version=1.22.22
+quarkus.quinoa.enable-spa-routing=true
+quarkus.quinoa.ui-root-path=/
+#quarkus.quinoa.ignored-path-prefixes=/saml,/oidc,/api,/ping
+#quarkus.quinoa.ignored-path-prefixes=/oidc
+quarkus.rest.path=/
+quarkus.log.level=DEBUG
```

Comment:
consider to restrict this

Reply:
`%dev.quarkus.log."it.pagopa.oneid".level=DEBUG`
could it be a good fit for us? Please note also the `%dev` prefix, which will set logging more verbose only for dev purpose

Reply:
lgtm!